src/styles/_boilerplate.scss

```diff
@@ -28,6 +28,7 @@ option {
 a,
 a:visited {
   text-decoration: none;
+  color: inherit;
```

Comment:
all good with auth forms after it? (signIn, signUp links)

Reply:
Remove this style for link!